addons/website_event/models/event_event.py

```diff
@@ -400,9 +400,9 @@ def _google_map_link(self, zoom=8):
 
     def _track_subtype(self, init_values):
         self.ensure_one()
-        if 'is_published' in init_values and self.is_published:
+        if 'website_published' in init_values and self.is_published:
```

Comment:
We could be include, and hook on both fields maybe ? Like
```suggestion
        if init_values.keys() & {'is_published', 'website_published'} and self.is_published:
```